3p-ecosystem/src/main/res/layout/fragment_commissionable.xml

```diff
@@ -42,18 +42,17 @@
 
         </com.google.android.material.appbar.AppBarLayout>
 
-        <androidx.recyclerview.widget.RecyclerView
-            android:id="@+id/listRecyclerView"
-            android:layout_width="0dp"
-            android:layout_height="0dp"
-            android:paddingVertical="@dimen/row_item_margin_vertical"
-            android:scrollbars="vertical"
-            app:layoutManager="LinearLayoutManager"
-            app:layout_constraintBottom_toBottomOf="parent"
-            app:layout_constraintEnd_toEndOf="parent"
-            app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toBottomOf="@+id/topBarLayout"
-            />
+                <!-- FIXME: the LazyColumn overlaps the top app bar
+                Tried a lot of things, but none worked.
+                Once we complete the full migration to Compose, having the topAppbar in Compose
+                will probably fix this issue, but until then, would be nice to have a fix.
+                -->
+                <androidx.compose.ui.platform.ComposeView
+                    android:id="@+id/compose_view"
+                    android:layout_width="match_parent"
+                    android:layout_height="match_parent"
+                    app:layout_constraintStart_toEndOf="@+id/topBarLayout"/>
+        <!--  FIXME: tried this as well:  app:layout_constraintTop_toBottomOf="@+id/topBarLayout"/> -->
```

Comment:
The `ComposeView` ought to respect the `ContraintLayout` layout params. Odd it wasn't.

Reply:
Agree. Leaving as is for now, the final migration PR should come soon, and it will take care of that.